src/util/hyper.rs

```diff
@@ -0,0 +1,154 @@
+///! Hyper compat layer after traits got split from tokio's
```

Comment:
Seems odd that we need this especially with the unsafe

Reply:
Will take a look, but that's how hyper does it for the examples now. It seems to be a recent, badly documented change. I will investigate more, hadn't seen the unsafe, was just trying to unblock myself from exploring the cli, my bad

Reply:
Ah, although they inline that logic in their examples and were recommending that in issues, apparently now it's part of hyper_util, so that should do it! https://docs.rs/hyper-util/latest/hyper_util/rt/tokio/struct.TokioIo.html

Reply:
Done, removed the inlining of that logic, using the one from hyper_util!